src/Security/Authenticator/BasicAuthenticator.php

```diff
@@ -14,6 +14,8 @@
 
 /**
  * Class BasicAuthenticator.
+ * @deprecated You should use the standard way of Symfony to do it. Learn more here:
+ *             https://symfony.com/doc/current/security/auth_providers.html#security-http-basic
```

Comment:
It would be great to mention it also in documentation ?

Reply:
My bad .. https://github.com/biig-io/UserComponent/pull/3/files#diff-361f92eed2eaa3203409b5ccbdfa0802R57